examples/kops-with-vpc/README.md

```diff
@@ -53,28 +53,63 @@ make plan
 make apply
 ```
 
-### Part 2: Kubernetes (Kops)
-
+### Part 2: Copy relevant values over to the Kubernetes' side
 ```
-popd; pushd kubernetes
+popd
+cp -r kubernetes/template kubernetes/$ENVIRONMENT
+make copy-keys                       # Copy generated .pem files
+make copy OUT=region AS=REGION       # Copy terraform output to kube(kops) environment: `$ENVIRONMENT/env`
```

Comment:
Could this whole list of variables be in the Makefile instead of needing to run each command separately?  One way would be for the Makefile to have a new phony target that runs each of these commands.